library/odoo

```diff
@@ -1,6 +1,6 @@
 Maintainers: Christophe Monniez <moc@odoo.com> (@d-fence)
 GitRepo: https://github.com/odoo/docker
-GitCommit: 26df6d6eef8df7a9927fbdade79772455de7e8eb
+GitCommit: ad549a7c3602557abc44e9c04369965a73d78fcd
 
 Tags: 15.0, 15, latest
 Architectures: amd64
```

Comment:
Tag 16 missing?

Reply:
Tags: 13.0, 13 also must be removed right @d-fence ?

Reply:
Yes, oversight